ooni/deck.py

```diff
@@ -85,11 +88,30 @@ def nettest_to_path(path, allow_arbitrary_paths=False):
     if allow_arbitrary_paths and os.path.exists(path):
         return path
 
-    fp = FilePath(config.nettest_directory).preauthChild(path + '.py')
-    if fp.exists():
-        return fp.path
-    else:
+    p = path.split("/")
+    test_name = p[0]
+    if len(p) > 1:
+        test_categories = [p[0]]
```

Comment:
This assignment is redundant because below you assign unconditionally to `test_categories`